src/components/auto-complete/auto-complete.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { OptionList } from '../selection-list/selection-list';
```

Comment:
the intention is for it to be common, and indeed i had plans to refactor this. should i do it in this PR? @alisey